extensions/winrt_smbus/winrt_smbus.py

```diff
@@ -0,0 +1,264 @@
+import _wini2c as i2c
+
+class SMBus:
```

Comment:
I'm not really familiar with SMBus. Looks like you're just using i2c. Are they the same?